.abs/main.yaml

```diff
@@ -1,5 +1,3 @@
-chart-dir: helm/ingress-nginx
```

Comment:
I think this and the line at the end of the file are mainly useful when executing tests outside of CI. I would vote to leave them in place

Reply:
Done :)